src/main/java/com/jaoafa/javajaotan2/command/Cmd_Alpha.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * jaoLicense
+ *
+ * Copyright (c) 2021 jao Minecraft Server
+ *
+ * The following license applies to this project: jaoLicense
+ *
+ * Japanese: https://github.com/jaoafa/jao-Minecraft-Server/blob/master/jaoLICENSE.md
+ * English: https://github.com/jaoafa/jao-Minecraft-Server/blob/master/jaoLICENSE-en.md
+ */
+
+package com.jaoafa.javajaotan2.command;
+
+import cloud.commandframework.Command;
+import cloud.commandframework.arguments.standard.StringArrayArgument;
+import cloud.commandframework.context.CommandContext;
+import cloud.commandframework.jda.JDACommandSender;
+import cloud.commandframework.meta.CommandMeta;
+import com.jaoafa.javajaotan2.lib.CommandPremise;
+import com.jaoafa.javajaotan2.lib.JavajaotanCommand;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.MessageChannel;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class Cmd_Alpha implements CommandPremise {
+    @Override
+    public JavajaotanCommand.Detail details() {
+        return new JavajaotanCommand.Detail(
+            "alpha",
+            Collections.emptyList(),
+            "アルファになったオレをします。",
+            Collections.emptyList()
+        );
```

Comment:
説明抜けで非常に申し訳ないんですが、

```suggestion
        return new JavajaotanCommand.Detail(
            "alpha",
            "アルファになったオレをします。"
        );
```
で動作します。

ちなみに多分permsは空リストだと誰も実行できなくなっちゃう恐れ。

参考: 
https://github.com/jaoafa/Javajaotan2/blob/52f8d8bcf3e0cbe5b86f9b5ffa5cec6d0242a1cd/src/main/java/com/jaoafa/javajaotan2/lib/JavajaotanCommand.java#L37-L63